### PR TITLE
Elements

### PR DESCRIPTION
**Pull request by shannonmoranetz**
**PR Summary**
- Added page elements to appropriate components
- Palettes (data array), currentPalette (id) added to global state
- Hex colors mapped out to ColorBox component as ColorCard components
- CSS: variables, main, and normalize moved to styles folder in /src

### CODE REVIEW

File: src/components/Header/Header.js [hunk 1]
```diff
@@ -1,9 +1,17 @@
 import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faFire } from '@fortawesome/free-solid-svg-icons';
```
Comment (hlhartley) [inline src/components/Header/Header.js:4]: Cool. I didn't know you could import the font awesome package and then use the icons as components themselves. Thanks for showing me this.

File: src/components/Header/Header.scss [hunk 2]
```diff
@@ -1,6 +1,32 @@
 .Header {
     align-items: center;
     display: flex;
-    height: 10vh;
+    flex-direction: column;
+    // height: 10vh;
```
Comment (hlhartley) [inline src/components/Header/Header.scss:5]: I think we can remove this ^

File: src/containers/App/App.js [hunk 3]
```diff
@@ -11,28 +13,44 @@ export class App extends Component {
     }
   }
 
-  componentDidMount = async () => {
+  componentDidMount = () => {
+    this.initialFetch();
+  }
+
```
Comment (hlhartley) [inline src/containers/App/App.js:19]: I like how you broke this out into a separate function

File: src/containers/ProjectList/ProjectList.scss [hunk 4]
```diff
@@ -2,4 +2,5 @@
     border: 2px solid tomato;
     display: flex;
     flex-direction: column;
+    overflow-y: scroll;
```
Comment (hlhartley) [inline src/containers/ProjectList/ProjectList.scss:5]: Can you show me how this works later?

File: src/components/Header/Header.scss (hunk 2 above)
Comment (shannonmoranetz) [reply src/components/Header/Header.scss:5]: Thank you! Removed in latest branch.

File: src/containers/App/App.js (hunk 3 above)
Comment (shannonmoranetz) [reply src/containers/App/App.js:19]: 🙇‍♀️

File: src/containers/ProjectList/ProjectList.scss (hunk 4 above)
Comment (shannonmoranetz) [reply src/containers/ProjectList/ProjectList.scss:5]: 👍

**Review by hlhartley — APPROVED** [review]
3/25: Reviewed in person.

**Pull request merged**